docs/CMakeLists.txt

```diff
@@ -40,8 +40,8 @@ set(COMPONENT_DIRS
 )
 set(MAINPAGE_FILE README.md)
 set(DOXYGEN_USE_MDFILE_AS_MAINPAGE ${MAINPAGE_FILE})
+set(PREDEFINED OLP_SDK_ENABLE_DEFAULT_CACHE)
```

Comment:
OLP_SDK_ENABLE_DEFAULT_CACHE is something totally different. Are you sure this is correct what you are doing or simply a copy-paste-error?

Reply:
Copy-paste error. Definitely. Already checked in the source files

Reply:
@nplakhtii, we agreed to first test changes to the CMake list locally and then create a CR. Therefore, this CR should not contain any changes in CMakeLists.txt. Please see this file in master (https://github.com/heremaps/here-data-sdk-cpp/blob/master/docs/CMakeLists.txt) and make sure your CR does not change it.

Reply:
sorry, my bad. removing this tag (will commit in another MR).